src/SPECS/hootenanny.spec.in

```diff
@@ -169,44 +169,263 @@ This package contains the UI and web services.
+        sudo -u postgres psql -d postgres -c "UPDATE pg_database SET datistemplate='true' WHERE datname='wfsstoredb'"
+        sudo -u postgres psql -d wfsstoredb -c 'create extension postgis;'
+        sudo -u postgres psql -d wfsstoredb -c "GRANT ALL on geometry_columns TO PUBLIC;"
+        sudo -u postgres psql -d wfsstoredb -c "GRANT ALL on geography_columns TO PUBLIC;"
+        sudo -u postgres psql -d wfsstoredb -c "GRANT ALL on spatial_ref_sys TO PUBLIC;"
+    fi
+
+    # configure Postgres settings
+    PG_HB_CONF=/var/lib/pgsql/$PG_VERSION/data/pg_hba.conf
+    if ! sudo grep -i --quiet hoot $PG_HB_CONF; then
+        sudo -u postgres cp $PG_HB_CONF $PG_HB_CONF.orig
+        sudo -u postgres sed -i '1ihost    all            hoot            127.0.0.1/32            md5' $PG_HB_CONF
+        sudo -u postgres sed -i '1ihost    all            hoot            ::1/128                 md5' $PG_HB_CONF
+    fi
+    POSTGRES_CONF=/var/lib/pgsql/$PG_VERSION/data/postgresql.conf
+    if ! grep -i --quiet HOOT $POSTGRES_CONF; then
+        sudo -u postgres cp $POSTGRES_CONF $POSTGRES_CONF.orig
+        sudo -u postgres sed -i s/^max_connections/\#max_connections/ $POSTGRES_CONF
+        sudo -u postgres sed -i s/^shared_buffers/\#shared_buffers/ $POSTGRES_CONF
+        sudo -u postgres bash -c "cat >> $POSTGRES_CONF" <<EOT
+#--------------
+# Hoot Settings
+#--------------
+max_connections = 1000
+shared_buffers = 1024MB
+max_files_per_process = 1000
+work_mem = 16MB
+maintenance_work_mem = 256MB
+checkpoint_segments = 20
+autovacuum = off
+EOT
+    fi
+    # configure kernel parameters
+    SYSCTL_CONF=/etc/sysctl.conf
+    if ! grep --quiet 1173741824 $SYSCTL_CONF; then
+        sudo cp $SYSCTL_CONF $SYSCTL_CONF.orig
+        echo "Setting kernel.shmmax"
+        sudo sysctl -w kernel.shmmax=1173741824
+        sudo sh -c "echo 'kernel.shmmax=1173741824' >> $SYSCTL_CONF"
+        #                 kernel.shmmax=68719476736
+    fi
+    if ! grep --quiet 2097152 $SYSCTL_CONF; then
+        echo "Setting kernel.shmall"
+        sudo sysctl -w kernel.shmall=2097152
+        sudo sh -c "echo 'kernel.shmall=2097152' >> $SYSCTL_CONF"
+        #                 kernel.shmall=4294967296
+    fi
+    sudo service postgresql-$PG_VERSION restart
+
+    # create the osm api test db
+    sudo -i /var/lib/hootenanny/scripts/SetupOsmApiDB.sh
+
+    # configure Tomcat environment
+    TOMCAT_ENV=/usr/sbin/tomcat6
+    if ! grep -i --quiet HOOT $TOMCAT_ENV; then
+        sudo cp $TOMCAT_ENV $TOMCAT_ENV.orig
+        echo "Configuring tomcat6 environment"
+        sudo bash -c "cat >> $TOMCAT_ENV" <<EOT
+#--------------
+# Hoot Settings
+#--------------
+HOOT_HOME=/var/lib/hootenanny
+LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib:$HOOT_HOME/lib:$HOOT_HOME/pretty-pipes/lib
+GDAL_DATA=/usr/local/share/gdal
+GDAL_LIB_DIR=/usr/local/lib
+HOOT_WORKING_NAME=hoot
+QT_HOME=/usr/local/qt-everywhere-opensource-src-4.8.6
```

Comment:
Those config settings came out of the https://github.com/ngageoint/hootenanny/blob/develop/scripts/InstallHoot-CentOS-RHEL.sh script, but maybe they are not correct when install gdal from rpm vs building from source.  And I guess the tests we run don't exercise any code that has hoot-services calling gdal.

I'll check the gdal spec file for correct paths.